### ISSUE TITLE
A way to exclude lemmas by default?

### ISSUE THREAD

**Issue by Einenlum** [simjanos-dev/LinguaCafe#92]
I think it could be nice to exclude some lemmas by default. I'm thinking about numbers and punctuation for example.

![n](https://github.com/simjanos-dev/LinguaCafe/assets/5675200/40fdfbac-da27-47ca-b642-e2183cd13b20)

**Comment by Einenlum** [comment]
Interesting. It seems like the search for this is `{"language":"german","term":"--"}`, which probably means it's already excluded in a way (I guess [here](https://github.com/simjanos-dev/LinguaCafe/blob/main/config/linguacafe.php#L142)?) but in the UI it appears as any other word that should be learned.

**Comment by simjanos-dev** [comment]
What do you mean by excluding them? Set as ignored by default instead of yellow? 

Ignoring numbers as well is a good idea.

**Comment by Einenlum** [comment]
> What do you mean by excluding them? Set as ignored by default instead of yellow?

Yes, it's quite confusing that from a UI perspective, they're considered as words I have to set a level for. I need to go one by one on each new special char and set them to "I know this", and it's counter intuitive to me.

**Comment by simjanos-dev** [comment]
I'll set them as ignored by default.

**Comment by simjanos-dev** [comment]
I've set words which are in the `words_to_skip` array to ignored automatically when they are imported, and also numbers. Unfortunately this will not apply for any word that you have already imported. 

Also a lot of languages have a large amount of unique symbols, so I've probably missed a lot. I've opened a new issue for this, and will add them in the future, but it is not a priority now. In the meantime, feel free to add any symbols to the `words_to_skip` config array if someone found more.
